webrecorder/browsertrix
# Add QA page analysis chart

Resolves https://github.com/webrecorder/browsertrix/issues/1589



### Changes
- Adds chart with page match stats to the crawl QA tab
- Updates "running" analysis status to include WACZ steps

Note, I deviated from the wireframes to remove the "QA Analysis" heading since we only have one card now. It also feels like "Page Match Analysis" is a more precise heading for this particular chart. Open to alternatives.

### Manual testing

1. Log in as crawler
2. Go to archived item without any QA runs. Verify that analysis chart is not visible in "Pages"
3. Go to archived item with a single QA run. Verify card and chart renders as expected
4. Choose a different QA run. Verify chart renders as expected

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Archived Item - QA - Partially completed QA run | <img width="1036" alt="Screenshot 2024-04-22 at 4 50 03 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/d21ebe88-72d6-4399-ac9f-09d5761174ab"> |
| Archived Item - QA - Completed QA run | <img width="1035" alt="Screenshot 2024-04-22 at 4 11 20 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/ccd56735-5a18-4a89-8235-cac3c287c390"> |
| Archived Item - QA - Viewing older run while QA is running | <img width="1038" alt="Screenshot 2024-04-22 at 4 18 44 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/c7aebf91-19c4-4105-b92c-2f6b2c58db39"> |
| Archived Item - QA - Info tooltip | <img width="341" alt="Screenshot 2024-04-22 at 4 11 39 PM" src="https://github.com/webrecorder/browsertrix/assets/4672952/7531fa1b-f574-4675-9954-5aef4afa3c87"> |


### Follow-ups

We'll probably want to refactor `btrix-meter` after conversation to `TailwindComponent` to simplify reusability.
